backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointService.java

```diff
@@ -54,6 +56,10 @@
 // TODO Needs documentation annotations
 public class EndpointService {
 
+    private static final List<EndpointType> systemEndpointType = List.of(
+            EndpointType.EMAIL_SUBSCRIPTION
```

Comment:
The same will happen with other Endpoints that are not meant to be created by the user, like the `Send to drawer`